apollo-compiler/src/main/kotlin/com/apollostack/compiler/TypeSpecBuilder.kt

```diff
@@ -0,0 +1,8 @@
+package com.apollostack.compiler
+
+import com.squareup.javapoet.TypeSpec
+
+interface TypeSpecBuilder {
```

Comment:
CodeGenerator is used everywhere by multiple code generator. I meant to have this interface to be used with the top-level spec builders, `OperationTypeSpecBuilder`, `FragmentTypeSpecBuilder`, `TypeDeclarationTypeSpecBuilder` on which the `write` method gets called.